src/task-dispatcher/common/system_info.cc

```diff
@@ -23,32 +23,41 @@ uint32_t td::system::num_logical_cores() noexcept { return std::thread::hardware
 uint32_t td::system::num_physical_cores() noexcept
 {
 #ifdef CC_OS_WINDOWS
-    cc::array<std::byte> buffer;
-    auto const to_ptr = [](std::byte* raw) { return reinterpret_cast<SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX*>(raw); };
-    ::DWORD length = 0;
+    std::byte buffer_stack[4096];
+    cc::array<std::byte> buffer_heap;
+    DWORD buffer_length = CC_COUNTOF(buffer_stack);

```

Comment:
hm, could we think about renaming CC_COUNTOF? it's a bit too short and ambiguous for my tastes. especially for something that is actually not used that often